### ISSUE TITLE
bandcamp-dl cannot be installed with setuptools >= 58.0.2 due to "demjson" dependency

### ISSUE THREAD

**Issue by valpogus** [Evolution0/bandcamp-dl#189]
**Describe the bug**
bandcamp-dl cannot be installed because the dependency "demjson" uses "use_2to3" during build, which is removed in setuptools >= 58.0.2.
* https://github.com/dmeranda/demjson/issues/40
* https://setuptools.pypa.io/en/latest/history.html#v58-0-0

**To Reproduce**
```
rm -rf ~/.cache/pip/
python3 -m pip install setuptools==58.0.2
python3 -m pip install bandcamp-downloader demjson==2.2.4
```
`error in demjson setup command: use_2to3 is invalid.`

**Expected behavior**
The package bandcamp-downloader is correctly installed.

**Desktop**
 - OS: Ubuntu
 - Version: Ubuntu 20.04.3
 - Python Version: 3.8.10

**Additional context**
Other projects depending on demjson are migrating to the new library "demjson3".
The library "demjson3" works only on Python 3 and using it would mean removing support for Python 2, but according to the changelog, support for Python 2 was "officially" removed on version 0.0.6.

**Comment by Evolution0** [comment]
Yep should just be a matter of switching the dependency and testing to make sure everything works but from the sound of it its a drop-in replacement.

**Comment by Evolution0** [comment]
Fixed in 0.0.11, pending release.

**Comment by rien333** [comment]
@Evolution0  Could you remove all references to demjson 2 from the project? The readme.md still links to the wrong page. Should be https://pypi.org/project/demjson3/.
